packages/connector-wechat-native/README.md

```diff
@@ -59,10 +63,33 @@ _Bundle ID_ and _Test version Bundle ID_ can be the same value.
 WeChat requires Universal Link for native sign-in. If you haven't set up or don't know it, please refer to the [Apple official doc](https://developer.apple.com/ios/universal-links/).
 :::
 
-Click "Submit Review" to continue.
+#### Android app (Android 应用)
+
+Check "Android app" (Android 应用).
+
+<img src="./docs/platform-android-app.png" alt="Android app platform" width="200px" />
```

Comment:
This screenshot should show more info to help developers locate the check box. With a similar scale as the previous iOS screenshot (in line 52).

Reply:
updated